docs/tutorials/migrating-from-docker-compose-to-kubernetes.md

```diff
@@ -0,0 +1,288 @@
+---
+layout: default
+title: "Migrating from Docker Compose to Kubernetes"
+permalink: /tutorials/migrating-from-docker-compose-to-kubernetes
+parent: Tutorials
+nav_order: 5
+---
+
+# Migrating from Docker Compose to Kubernetes
+
+## Prerequisites
+
+Complete the [Migration Workflow](/tutorials/migration-workflow) tutorial first and familiarize yourself with the [transformation](/tutorials/migration-workflow/transform) process.
+
+## Overview
+
+In this tutorial we will look at migrating an application written for Docker Compose to run on Kubernetes.
+
+We will be using the 2 [Docker Compose](https://github.com/konveyor/move2kube-demos/tree/076850e75846a11b58fcfa12aeb49468ca045423/samples/docker-compose) samples from the [move2kube-demos](https://github.com/konveyor/move2kube-demos) repo.
+
+The [first sample](https://github.com/konveyor/move2kube-demos/tree/076850e75846a11b58fcfa12aeb49468ca045423/samples/docker-compose/single-service) is a web app with a single service using Nginx. It uses a prebuilt image.
```

Comment:
Do we want to mention about single-service one or remove it? the steps might be slightly different since the build is not there in single service?

Reply:
Let's retain it, the steps are not that different.